### DISCUSSION TITLE
ValueError: The window size must be less than or equal to 1

### DISCUSSION THREAD

**Discussion by adityabhandwalkar** [TDAmeritrade/stumpy#757] [Help: Coding & Implementations] [answered]
Hello ,

I am getting this error while running the code where I'm using the mass 
Here my main series is T = [2.96894023e+02  6.13719322e-01  4.84978187e-01  ...  9.13753619e-02   4.18911267e-02  7.85543579e-02]

and query is  original_y = np.array(original_y.astype(np.float64)) which 

original_y =[[0.79468292]
 [1.26130937]
 [1.08211665]
 ...
 [0.10033153]
 [0.04561889]
 [0.04183081]]
`query_dist = stumpy.mass(original_y,data)`
like this but I'm getting the error above , How can I solve this error?

**Comment by seanlaw** [comment, marked as answer]
@adityabhandwalkar The issue is that the query needs to be a 1-dimensional array and you've provided a 2-dimensional array to `stumpy.mass`.
